partner_second_lastname/views/+

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Delete this file

Reply:
Why?

Reply:
Done @emagdalenaC2i 

Thanks

Reply:
The file named "partner_second_lastname/views/+" because it is not used and I think is the same as "partner_second_lastname/views/res-partner.xml"